lib/ansible/plugins/lookup/password.py

```diff
@@ -110,7 +110,7 @@
 
 - name: create random but idempotent password
   ansible.builtin.set_fact:
-    password: "{{ lookup('ansible.builtin.password', '/dev/null', seed=inventory_hostname) }}"
+    password: "{{ lookup('ansible.builtin.password', '/dev/null seed=' + inventory_hostname) }}"
```

Comment:
this is incorrect, this example is to specifically show the comma separated method of passing arguments